tensorflow/contrib/losses/python/losses/loss_ops.py

```diff
@@ -104,7 +78,8 @@ def _safe_mean(losses, num_present):
       then zero is returned.
   """
   total_loss = math_ops.reduce_sum(losses)
-  return _safe_div(total_loss, num_present)
+  return math_ops.div_no_nan(total_loss, num_present,
+                             negative_to_zero=True, name="value")
```

Comment:
I'd much prefer to replace `div_no_nan(x, y, negative_to_zero=True, ...)` with `div_no_nan(x, tf.max(y, 0), ...)` and not add the `negative_to_zero` argument, which I doesn't look to me like it makes things simpler.

Reply:
Reasonable. I'd like to revise code as suggested later.